plat/kvm/x86/entry64.S

```diff
@@ -40,6 +40,8 @@
 #define MYMULTIBOOT_FLAGS \
     (MULTIBOOT_PAGE_ALIGN | MULTIBOOT_MEMORY_INFO | MULTIBOOT_AOUT_KLUDGE)
 
+#ifndef CONFIG_ENABLE_PIE
+
```

Comment:
Instead of modifying the KVM platform that still can't be booted with this change, I would disable the KVM platform as soon PIE is enabled. As long as we do not have a first stage ASLR bootloader for KVM I would consider the platform incompatible. Linuxu should work with PIE, do you know if Xen works?